### ISSUE TITLE
Cannot find TB 68 extension from within Thunderbird

### ISSUE THREAD

**Issue by keunes** [conema/monterail-dark#35]
Hi,

I don't know why, but I can't find the extension that was provided (through #31) from within Thunderbird:
![Screenshot from 2019-12-16 07-28-58](https://user-images.githubusercontent.com/11229646/70884468-69665d00-1fd6-11ea-9023-42a0fd4eb9e8.png)

And when disabling the version filters:
![Screenshot from 2019-12-16 07-34-32](https://user-images.githubusercontent.com/11229646/70884581-a92d4480-1fd6-11ea-9c9e-b29bcf0358d5.png)

Two things: should we enquire with the Thunderbird people why this might be happening? And can you add a link to the new addon to the old addon's description?

Thanks!

**Comment by keunes** [comment]
Actually, @conema, might it be a setting on your side (at the Thunderbird repository)? It doesn't show up on your profile either (though through a direct link I can visit the page which does seem linked to your profile).

**Comment by conema** [comment]
Hi @keunes, it doesn't show because the addon is pending review by the Thunderbird admins and it will not show until it is reviewed & accepted. Sadly, I uploaded the theme ~1 month ago and I'm still waiting... Things seem a little stuck.

Anyway, the link is already present in the old addon page:
![immagine](https://user-images.githubusercontent.com/12801153/70885925-8b61de80-1fda-11ea-950f-7d95d50d5af5.png)

**Comment by keunes** [comment]
Thanks for adding that link, and for the quick reply! Unfortunately when opening the page, as it's unreviewed, it still can't be added with the click of a button (only by downloading and manually importing the file). Oh well, I have it now. But I would hope it'd be visible for others also as it's a great addition to Thunderbird.

I had a read on the [Mozilla forums](https://discourse.mozilla.org/search?q=review%20category%3A111), and I'm wondering now: what's the queue position, and is the add-on progressing a bit? I guess creating a post there won't help much, or maybe it does when a deprecated add-on is concerned?

**Comment by conema** [comment]
I posted in the [topicbox](https://thunderbird.topicbox.com/groups/addons?subscription_form=8c2eaa9a-c0eb-11e8-bb0b-59b48f05e7cf) for add-ons developers and I'm waiting for a reply. Like other people in the Mozilla Discourse, the queue is stuck and currently the theme is the 5th.
I hope that one reviewer sees the post and accept the add-on, I saw that for the other people that wrote in the forum, the add-on was accepted a few days later. I'll update you.
